frontend/tampere/enCustomizations.tsx

```diff
@@ -474,133 +474,136 @@ const en: DeepPartial<Translations> = {
-        sivustolla ongelman, joka ei ole luettelossa, otathan meihin yhteyttä.
+        Users may still encounter some issues on the website. The following
+        contains a description of known accessibility issues. If you notice an
+        issue on the site that is not listed, please contact us.
       </P>
       <ul>
         <li>
-          Viestit-sivulla liikkuminen näppäimistöllä tai ruudunlukijalla vaatii
-          vielä korjauksia siirtymien ja kohdistettavien elementtien osalta.
+          Navigating on the Messages page using the keyboard or screen reader
+          still requires revision for part of moving and targeted elements.
         </li>
         <li>
-          Palvelun yksikkökartassa ei pysty liikkumaan
-          näppäimistöllä/ruudunlukijalla, mutta yksikköjä voi selata samassa
-          näkymässä olevalta listalta. Palvelussa käytetty kartta on kolmannen
-          osapuolen tuottama.
+          The service’s unit map cannot be navigated using the keyboard/screen
+          reader, but the units can be browsed on the list available in the same
+          view. The map used in the service is produced by a third party.
         </li>
       </ul>
-      <H2>Kolmannet osapuolet</H2>
+      <H2>Third parties</H2>
       <P>
-        Verkkopalvelussa käytetään seuraavia kolmannen osapuolen palveluita,
-        joiden saavutettavuudesta emme voi vastata.
+        The online service uses the following third party services, the
+        accessibility of which we cannot be responsible for.
       </P>
       <ul>
-        <li>Keycloak käyttäjän tunnistautumispalvelu</li>
-        <li>Suomi.fi-tunnistautuminen</li>
-        <li>Leaflet-karttapalvelu</li>
+        <li>Keycloak user identification service</li>
+        <li>Suomi.fi identification</li>
+        <li>Leaflet map service</li>
       </ul>
-      <H2>Vaihtoehtoiset asiointitavat</H2>
+      <H2>Alternative ways of accessing the service</H2>
       <P>
-        <strong>Varhaiskasvatuksen ja esiopetuksen asiakaspalvelu</strong>
+        <strong>
+          Early childhood and pre-primary education customer service
+        </strong>
         <br />
-        sähköposti{' '}
+        email{' '}
         <a href="mailto:varhaiskasvatus.asiakaspalvelu@tampere.fi">
           varhaiskasvatus.asiakaspalvelu@tampere.fi
         </a>
         <br />
-        puhelin <a href="tel:040 800 7260">040 800 7260</a>, soittoaika ma–pe
-        klo 9.00–12.00
+        telephone <a href="tel:040 800 7260">040 800 7260</a>, calling hours
+        Mon-Fri 9.00–12.00
       </P>
-      <H2>Anna palautetta</H2>
+      <H2>Give feedback</H2>
       <P>
-        Jos huomaat saavutettavuuspuutteen verkkopalvelussamme, kerro siitä
-        meille.{' '}
+        If you notice an accessibility gap in our online service, please let us
+        know! You can give us feedback using the{' '}
         <ExternalLink
           href="https://elomake.tampere.fi/lomakkeet/21932/lomakkeet.html"
```

Comment:
Is there an English version of this form? Seems counter productive to link to a purely Finnish form.